lib/cosmos/top_level.rb

```diff
@@ -428,6 +428,7 @@ def self.create_log_file(filename, log_dir)
             yield file
           ensure
             file.close unless file.closed?
+            File.chmod(0444, log_file) # Make file read only
```

Comment:
See the comment above.   I think this could cause issues with the same file getting opened twice to capture two exceptions in the same second.

Reply:
What do you propose? Ultimately I think exception files should be read-only once they're finished.

Reply:
Do we add milliseconds to the exception filename?

Reply:
Need to a add a wait until the timestamp changes if the file already exists.   Adding milliseconds would make that wait smaller....

Reply:
I didn't want to change our logfile format by adding milliseconds. Since the odds of calling this twice in one second are low I think that's ok.